recipe/build.sh

```diff
@@ -1,48 +1,27 @@
 #!/bin/bash
+
+set -e
+
 # Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.guess config.fsf.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.sub config.fsf.sub
+cp "${BUILD_PREFIX}"/share/gnuconfig/config.guess config.fsf.guess
+cp "${BUILD_PREFIX}"/share/gnuconfig/config.sub config.fsf.sub
 
 shopt -s extglob
 chmod +x configure
 
+export PKG_CONFIG_LIBDIR=${PREFIX}/lib/pkgconfig:${PREFIX}/share/pkgconfig
+
 mkdir build
 cd build
 
-if [[ "$target_platform" == "linux-ppc64le" ]]; then
-  # HOST="powerpc64le-conda-linux-gnu" masks the fact that we are only
-  # building for power8 and uses an older POWER architecture.
-  GMP_HOST="power8-pc-linux-gnu"
-else
-  GMP_HOST=$HOST
-fi
+GMP_HOST="${HOST}"
 
-../configure --prefix=$PREFIX --enable-cxx --enable-fat --host=$GMP_HOST
+../configure --prefix="${PREFIX}" --enable-cxx --enable-fat --host="${GMP_HOST}"
 
-make -j${CPU_COUNT} ${VERBOSE_AT}
-if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
```

Comment:
Removing things from conda-=forge IMo makes our recipes harder to maintain.

Reply:
As we do not provide cross-compilation we can remove it. 
It's not harder because it's not the code we run on our CI.

Reply:
Removing stuff from our recipe just "because" makes it harder to later sync our recipes with conda-forge.

Reply:
Fixed